ndb/src/google/cloud/ndb/_runstate.py

```diff
@@ -22,6 +22,12 @@
 
 class State:
     eventloop = None
+    credentials = None
+    host = None
+    namespace = None
+    stub = None
+    secure = False
+    project = None
```

Comment:
Why not put these all in a constructor as optional keywords? I ask because they are lowercase class attributes (similar to the `ndb` pattern in `Property` for fallback defaults, but not particularly great is it?).